cabinet/cabinet.py

```diff
@@ -56,6 +56,20 @@ def get_all(self):
     def get(self, name):
         return self._vault.get(name)
 
+    def get_by_tags(self, tags=[]):
+        """
+        Recover all the items that contains the given tags.
+
+        @param tags: A list of tags
```

Comment:
Indeed! Here there is another line within the same code in master with `@param`:
https://github.com/cabinet/cabinet-lib/blob/master/cabinet/cabinet.py#L62

I'll update both to `:param`.